addons/service/downloadmanager/FlexGet/source/system.d/service.downloadmanager.FlexGet.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=FlexGet
+After=graphical.target
+
+[Service]
+Type=idle
+ExecStart=/bin/sh -c "exec sh /storage/.kodi/addons/service.downloadmanager.FlexGet/bin/FlexGet.start daemon start"
+ExecStop=/bin/sh -c "exec sh /storage/.kodi/addons/service.downloadmanager.FlexGet/bin/FlexGet.start daemon stop"
```

Comment:
no need for this. just make sure systemd knows daemon's mainpid (best to use "exec python ... ..." in your start script) and systemd will take care to kill mainpid and (if any) subprocesses